pkg/util/metric/prometheus_exporter.go

```diff
@@ -125,6 +126,9 @@ func (pm *PrometheusExporter) ScrapeRegistry(registry *Registry, includeChildMet
 func (pm *PrometheusExporter) printAsText(w io.Writer, contentType expfmt.Format) error {
 	enc := expfmt.NewEncoder(w, contentType)
 	for _, family := range pm.families {
+		if left, _, found := strings.Cut(*family.Help, "\n"); found {
```

Comment:
Would it make more sense to terminate at the first `.` instead of `\n`? That will guarantee that the sentence is complete.

Reply:
Thanks for the nudge. I took a different approach that cleans metrics up when they get added to a registry. This will clean up the whitespace (newlines/indents) and truncate at the first sentence.